domains/POAS/events/lpa-access-granted/schema.json

```diff
@@ -0,0 +1,28 @@
+{
+  "$id": "https://opg.service.justice.gov.uk/opg.poas.use/lpa-access-granted.json",
+  "$schema": "https://json-schema.org/draft/2020-12/schema",
+  "title": "opg.poas.use/lpa-access-granted",
+  "type": "object",
+  "properties": {
+    "uid": {
+      "type": "string",
+      "description": "The UID of the LPA",
+      "pattern": "^M(-[A-Z0-9]{4}){3}$"
+    },
+    "lpaType": {
+      "type": "string",
+      "description": "The type of LPA",
+      "enum": ["pfa", "hw", "personal-welfare", "property-and-affairs"]
```

Comment:
This should probably only be the modernised ["personal-welfare", "property-and-affairs"] as there is no HW or PFA for modernised LPAs.